package/Dockerfile

```diff
@@ -15,12 +15,14 @@ WORKDIR /var/lib/rancher
 
 ARG ARCH=amd64
 ARG IMAGE_REPO=rancher
+# System-charts will continue to point to dev-v2.6 branch since eventually in 2.7.x the plan is
+# to deprecate the v1 tools and remove this dependency
```

Comment:
I'd update to use the language "remove the v1 tools"

Reply:
updated